doc/krillta.1

```diff
@@ -0,0 +1,26 @@
+.TH "krillta" "1" "NLnet Labs"
+.SH NAME
+krillta - Krill Offline Trust Anchor
+.SH SYNOPSIS
+krillta [SUBCOMMAND]
+
+krillta help [SUBCOMMAND]
+
+krillta -h, --help
+
+krillta -V, --version
+.SH DESCRIPTION
+Krill is a free, open source RPKI Certificate Authority that lets you run
+delegated RPKI under one or multiple Regional Internet Registries (RIRs). 
+Through its built-in publication server, Krill can publish Route Origin
+Authorizations (ROAs) on your own servers or with a third party.
+
+The krillta tool can be used to set up Krill to operate an RPKI Trust
+Anchor (TA). An RPKIA TA serves as an entrypoint for RPKI validators. If
+you are not an RIR, then you will not need to run your on RPKI TA for
+normal RPKI operations.
+
+For more information visit https://rpki.readthedocs.io/en/latest/krill/trust-anchor.html.
```

Comment:
The first paragraph is not that interesting to TA users - which would be power users - and the documentation link is wrong. I suggest:

The krillta tool is used for managing a Krill based RPKI Trust Anchor.

For more information visit https://krill.docs.nlnetlabs.nl/en/latest/trust-anchor.html